api/src/main/resources/workbench-api.yaml

```diff
@@ -4503,6 +4521,27 @@ paths:
           description: Handle Free Credits Expiry Task Ran Successfully
           content: { }
       x-codegen-request-body-name: request
+  /v1/cloudTask/checkCreditsExpirationForUserIDs:
+    post:
+      tags:
+      - cloudTaskUser
+      - cloudTask
+      description: |
+        Run the business logic to handle users who have consumed or about to consume their free credits. Including disabling their workspaces and sending them notifications.
```

Comment:
I'm not sure "disabling their workspaces" is precise.  Users can still do things like edit descriptions and create cohorts.  "Disable compute in workspaces" ?

Also add "delete cloud environments" or similar text.